app/src/main/java/org/simple/clinic/editpatient/EditPatientEffectHandler.kt

```diff
@@ -203,7 +203,7 @@ class EditPatientEffectHandler @AssistedInject constructor(
   private fun coerceAgeFrom(recordedAgeDetails: PatientAgeDetails, enteredAge: String): PatientAgeDetails {
     val enteredAgeValue = enteredAge.toInt()
     return when {
-      recordedAgeDetails.type == FROM_AGE && recordedAgeDetails.ageValue == enteredAgeValue -> recordedAgeDetails
```

Comment:
We will be removing the types usage entirely, right?

Reply:
Yeah, that is the eventual plan.